app/build.gradle

```diff
@@ -45,8 +46,8 @@ android {
         applicationId "com.hover.stax"
         minSdk 21
         targetSdk 32
-        versionCode 199
-        versionName "1.18.0"
+        versionCode 183
```

Comment:
Please increase the version information

Reply:
Will leave this as the last thing, incase anyone merges anything before this gets in